guides/run-full-node.md

```diff
@@ -260,42 +260,52 @@ In the service file you created, replace any occurrences of `/home/digibyte/digi
 
 When the time for upgrade comes, stop the `digibyted.service`, download and decompress the new release, delete and re-create the symbolic link pointing to the new DigiByte directory and start the `digibyted.service`.
 
-# Raspberry Pi 4
+# Raspberry Pi
 
-> It's always recommended to have active fan cooling on your Raspberry Pi 4.
+> You can also use [DigiNode Tools](https://diginode.tools/) to setup and manage a DigiByte Node on a Raspberry Pi.
+{.is-info}
+
+> A Raspberry Pi 4 or later is required. Raspberry Pi 3 and older are not supported. It's always recommended to have active fan cooling on the Raspberry Pi.
 {.is-info}
 
 
 ## Prerequisites
 
 | Requirements | Minimum | Recommended |
 | --- | --- | --- |
+| Hardware | Raspberry Pi 4 4Gb | Raspberry Pi 5 8Gb or better |
```

Comment:
Generally speaking this is definitely reasonable. Some people like to tinker with their Raspberry Pi chips however to achieve a significantly increased throughput, enabling even older models to reach the same capacity as a Raspberry Pi 4. That said, I think it's fair to generalize here.